content/articles/2018-10-devtools-2.0.0.Rmarkdown

```diff
@@ -0,0 +1,256 @@
+  author: Hunter Haley
+---
+
+## Introduction
+
+[devtools] 2.0.0 is now on CRAN!
+
+devtools makes package development easier by providing R functions that
+simplify and expedite common tasks. [R Packages] is a book based around this
+workflow.
+
+devtools 2.0.0 is a major release that contains work from the past year and a
+half, since the major devtools release (1.13.0).
+
+## Breaking changes
+
+There are a handful of breaking changes in this release, mainly done in the
+interest of simplifying and standardizing the function APIs.
+
+The changes should affect relatively little user code, but have required
+developers to make changes. Common errors and ways to work around them
+as well as the full list of changes can be found in the [Breaking
+changes](https://github.com/r-lib/devtools/blob/master/NEWS.md#breaking-changes)
+section of devtools'
+[NEWS](https://github.com/r-lib/devtools/blob/master/NEWS.md). If you
+discover something missing, please let us know so we can add it.
+
+## Conscious uncoupling
+
+This release splits the functionality in devtools into a number of smaller
+packages which are simpler to develop and also easier for other packages to
+depend on. In particular the following packages have been spun off in what we
+are calling the [conscious uncoupling] of devtools.
+
+* [remotes](https://remotes.r-lib.org): Installing packages (i.e.
+  `install_github()`).
+* [pkgbuild](https://github.com/r-lib/pkgbuild): Building binary packages
+  (including checking if build tools are available) (i.e. `build()`).
+* [pkgload](https://github.com/r-lib/pkgload): Simulating package loading (i.e.
+  `load_all()`).
+* [rcmdcheck](https://github.com/r-lib/rcmdcheck): Running R CMD check and
+  reporting the results (i.e. `check()`).
+* [revdepcheck](https://github.com/r-lib/revdepcheck): Running R CMD check on
+  all reverse dependencies, and figuring out what's changed since the last CRAN
+  release (i.e. `revdep_check()`).
+* [sessioninfo](https://github.com/r-lib/sessioninfo): R session info (i.e.
+  `session_info()`).
+* [usethis](https://usethis.r-lib.org): Automating package setup (i.e.
+  `use_test()`).
+
+devtools will remain the main package developers will interact with when
+writing R packages; it will just rely on these other packages internally
+for most of the functionality.
+
+Generally, you should not need to worry about these different packages, because
+devtools installs them all automatically. You will need to care, however, if
+you're filing a bug because reporting it at the correct place will lead to a
+speedier resolution.
+
+Package developers who wish to depend on devtools features should also pay
+attention to which package the functionality is coming from and depend on that
+rather than devtools. In most cases packages should not depend on devtools
+directly. This is similar to the situation with the [tidyverse
+package](https://www.tidyverse.org/articles/2018/06/tidyverse-not-for-packages/).
```

Comment:
Good point to raise. Given that you raise it, do you think there should be any explanation about the different implementations used in tidy verse vs. devtools in terms of offering a meta-package? I mean, devtools remains much more of a real package than tidyverse is.